src/q7/matrix-INMA2380/exercises/ch2.tex

```diff
@@ -1,17 +1,38 @@
 \section{Linear system}
-\exo{0}
-Verify that
-$\mathcal{S}_1 \cap \mathcal{S}_2$,
-$\mathcal{S}_1 + \mathcal{S}_2$,
-$\Ker A$, $\Ima A$ and $A \mathcal{S}$
-are linear subspaced.
-\nosolution
+\exo{1}
```

Comment:
I don't remember why I used `0` as exercise number and included the statement. Any idea why and why it needs to change ?

Reply:
Ahah ok makes sense :-D